database/migrations/000006_create_transactions_to_blocks_table.up.sql

```diff
@@ -1,14 +1,15 @@
-CREATE TABLE `transactions_to_blocks`
+CREATE TABLE transactions_to_blocks
 (
-    `transaction_id`    BIGINT UNSIGNED NOT NULL,
-    `block_id`          BIGINT UNSIGNED NOT NULL,
-    `index` INT UNSIGNED    NOT NULL,
-    PRIMARY KEY (`transaction_id`, `block_id`),
-    INDEX `idx_transactions_to_blocks_index` (`index`),
-    CONSTRAINT `fk_transactions_to_blocks_block_id`
-        FOREIGN KEY (`block_id`)
-            REFERENCES `blocks` (`id`),
-    CONSTRAINT `fk_transactions_to_blocks_transaction_id`
-        FOREIGN KEY (`transaction_id`)
-            REFERENCES `transactions` (`id`)
+    transaction_id BIGINT NOT NULL,
+    block_id       BIGINT NOT NULL,
+    index BIGINT CHECK (index >= 0) NOT NULL,
```

Comment:
index is uint32, originally, so needs to make sure values stay within uint32 range